scripts/eintsgit.py

```diff
@@ -208,34 +208,41 @@ def eints_download(settings, credits_file):
     )
 
 
-def update_eints_from_git(settings, force):
+def update_eints_from_git(settings, force, pull):
     """
     Perform the complete operation from syncing Eints from the repository.
 
     @param force: Upload even if no changes.
     @type  force: C{bool}
+
+    @param pull: Pull from remote, or use working copy as-it.
+    @type  pull: C{bool}
     """
 
     with FileLock(lock_file):
         print_info("Check updates from git")
-        if git_pull(settings) or force:
+        if pull and git_pull(settings) or force:
```

Comment:
Sorry, I was wrong. Mobile interface was playing tricks with me. Didn't spot that there was both `and` and `or`.

Can I suggest we rework this function a bit, as it feels a bit awkward now.

```
upload = force
If pull and git_pull .. 
  Upload = True
...
```

(Still on mobile, so typos are expected)